docs/4.0/getting-started/accessibility.md

```diff
@@ -10,25 +10,25 @@ Bootstrap provides an easy-to-use framework of ready-made styles, layout tools,
 
 ## Overview and Limitations
 
-The overall accessibility of any project built with Bootstrap depends in large part on the author's markup, additional styling, and scripting they've included. However, provided that these have been implemented correctly, it should be perfectly possible to create websites and applications with Bootstrap that fulfill [<abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.0](https://www.w3.org/TR/WCAG20/) (A/AA/AAA), [Section 508](https://www.section508.gov/) and similar accessibility standards and requirements.
+The overall accessibility of any project built with Bootstrap depends in large part on the author's markup, additional styling, and included scripts. However, provided that these have been implemented correctly, it should be perfectly possible to create websites and applications with Bootstrap that fulfill [<abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.0](https://www.w3.org/TR/WCAG20/) (A/AA/AAA), [Section 508](https://www.section508.gov/) and similar accessibility standards and requirements.
```

Comment:
changing from "scripting they've included" to "included scripts" changes the meaning, i'd say. "included scripts" may be mistaken for scripts included by Bootstrap, while we tried to convey here that it's about scripts that are not part of Bootstrap that authors have added on top of what's provided here.